beacon-chain/blockchain/process_block.go

```diff
@@ -144,19 +144,14 @@ func getStateVersionAndPayload(st state.BeaconState) (int, interfaces.ExecutionD
 	return preStateVersion, preStateHeader, nil
 }
 
-func (s *Service) onBlockBatch(ctx context.Context, blks []interfaces.ReadOnlySignedBeaconBlock,
-	blockRoots [][32]byte) error {
+func (s *Service) onBlockBatch(ctx context.Context, blks []consensusblocks.ROBlock) error {
```

Comment:
this is not related to this PR, but I would prefer the type to signal that the block is signed.

Reply:
maybe this can be another PR against `develop` if we decide to go with RO block route here